src/roiextractors/testing.py

```diff
@@ -203,7 +203,11 @@ def check_segmentations_equal(
     ) == set(
         segmentation_extractor2.get_roi_pixel_masks(roi_ids=segmentation_extractor1.get_roi_ids()[:1])[0].flatten()
     )
-    assert_array_equal(segmentation_extractor1.get_image(), segmentation_extractor2.get_image())
```

Comment:
Maybe you can test for the length of the dictionary being equal, if that does not work then the other test is not necesssary. Also, maybe you can check for the names of the images being equals so the `get_images_dict()`  are compared both in keys and items?

Reply:
I moved this to a separate check method, thanks for the suggestions.